### ISSUE TITLE
Deb packaging issue in ppc64le

### ISSUE THREAD

**Issue by qhaas** [VirtualGL/virtualgl#116]
While virtualgl builds (and 'works') in the ppc64le flavor of Ubuntu 18.04 on a power9 system using `make deb`, the packaging scripts (reasonably) assume the architecture is known as 'ppc64le' (which is what is returned by `uname -m`), however on [debian based systems](https://wiki.debian.org/ppc64el#Sbuild_on_ppc64el), it is actually 'ppc64el', which results in this error when trying to install the resulting deb package:
```
# apt-get install -f ./virtualgl_2.5.2_ppc64le.deb
...
dpkg: error processing archive /root/virtualgl-2.5.2/build/virtualgl_2.5.2_ppc64le.deb (--unpack):
 package architecture (ppc64le) does not match system (ppc64el)
Errors were encountered while processing:
 /root/virtualgl-2.5.2/build/virtualgl_2.5.2_ppc64le.deb
```

A workaround for end-users is to [add 'ppc64le' architecture](https://askubuntu.com/a/752323):
```
# dpkg --add-architecture ppc64le
```

**Comment by dcommander** [comment]
Do Debian-based systems always use ppc64el rather than ppc64le for the package architecture? Or does it vary? Just trying to figure out why the package architecture differs from `uname -m`, since this seems like the only architecture for which that’s the case.

**Comment by qhaas** [comment]
> Do Debian-based systems always use ppc64el rather than ppc64le for the package architecture? 

I believe this is the case, found Debian's choice of 'el' over 'le' rather inconsistent with the rest of the ecosystem.  Both Debian Stable and Ubuntu 18.04 LTS use 'el':
```
# podman run --rm ppc64le/ubuntu:18.04 apt list --installed 2> /dev/null | grep ppc64el | wc -l
79
# podman run --rm ppc64le/debian:stable apt list --installed 2> /dev/null | grep ppc64el | wc -l
82
# podman run --rm ppc64le/ubuntu:18.04 apt list --installed 2> /dev/null | grep ppc64le | wc -l
0
# podman run --rm ppc64le/debian:stable apt list --installed 2> /dev/null | grep ppc64le | wc -l
0
```
EL7 based ppc64le distributions stick to 'ppc64le', consistent with `uname -m` on this platform.
```
# podman run --rm ppc64le/centos:7 yum list installed | grep ppc64el | wc -l
0
# podman run --rm ppc64le/centos:7 yum list installed | grep ppc64le | wc -l
132
```

**Comment by dcommander** [comment]
I tried to reproduce this problem but couldn't.  VirtualGL's packaging system seems to properly set the .deb architecture to ppc64el.
https://github.com/VirtualGL/virtualgl/blob/master/cmakescripts/BuildPackages.cmake#L53-L54

**Comment by dcommander** [comment]
Ah, I think I see the problem.  The fix for this was in 5fa909ff0, which was introduced in VirtualGL 2.6, but you're trying to build 2.5.2, which is outdated.

**Comment by qhaas** [comment]
Doh!  I should have updated first before complaining, was running 2.5.2 to match the version that ships in the EL7 repository.  Thanks!
